### PR TITLE
[pwrmgr,dv] Sign off V2S

### PR DESCRIPTION
**Pull request by jdonjdon**
post review actions items(#12075) have been resolved by
#12129
#12758
#12831
#15001
#15025 
#15124 

Signed-off-by: Jaedon Kim <jdonjdon@google.com>

### CODE REVIEW

**Comment by tjaychen** [comment]
do we need to hold off on officially signing off v2s until some of the issues related to the fault register change are addressed?

**Comment by jdonjdon** [comment]
> 

Are you referring to this? #14785

**Comment by tjaychen** [comment]
> > 
> 
> Are you referring to this? #14785

sorry i was referring to #13950.

we can still merge this PR though, I just meant final closure might need to wait until the dv fallout from that has been handled.

**Review by matutem — APPROVED** [review]
LGTM, and thanks!

**Comment by jdonjdon** [comment]
I will merge this PR with #14785 and #15203

File: hw/ip/pwrmgr/doc/checklist.md [hunk 1]
```diff
@@ -221,11 +221,11 @@ Review        | [V3_CHECKLIST_SCOPED][]                 | Done        |
 
  Type         | Item                                    | Resolution  | Note/Collaterals
 --------------|-----------------------------------------|-------------|------------------
-Documentation | [SEC_CM_TESTPLAN_COMPLETED][]           | Not Started |
-Tests         | [FPV_SEC_CM_VERIFIED][]                 | Not Started |
-Tests         | [SIM_SEC_CM_VERIFIED][]                 | Not Started |
-Coverage      | [SIM_COVERAGE_REVIEWED][]               | Not Started |
-Review        | [SEC_CM_DV_REVIEWED][]                  | Not Started |
+Documentation | [SEC_CM_TESTPLAN_COMPLETED][]           | Done        |
+Tests         | [FPV_SEC_CM_VERIFIED][]                 | NA          |
```
Comment (cindychip) [inline hw/ip/pwrmgr/doc/checklist.md:225]: why is this N/A? I think we have pwrmgr fpv right?

Comment (cindychip) [reply hw/ip/pwrmgr/doc/checklist.md:225]: https://reports.opentitan.org/hw/top_earlgrey/formal/summary/latest/report.html
pwrmgr sec_cm is 100%, i think you can mark it as done

Comment (jdonjdon) [reply hw/ip/pwrmgr/doc/checklist.md:225]: You are right. I will update.

**Comment by jdonjdon** [comment]
> I will merge this PR with #14785 and #15203

Both PR's are merged. I am merging it now.
@matutem @tjaychen 
Feel free to re-open if I miss anything.

**Comment by tjaychen** [comment]
hey all, i think we won't be able to officially move pwrmgr to v2s yet until the reset domain issue is fixed.

**Comment by jdonjdon** [comment]
> 

Oh, are you expecting pwrmgr design change for reset domain issue?

**Comment by tjaychen** [comment]
> > 
> 
> Oh, are you expecting pwrmgr design change for reset domain issue?

yes sorry. This change will impact rstmgr / pwrmgr / pinmux. 
The new ndm reset will have to behave more like a hw_reset, and that will probably need some changes on `pwrmgr` side.

@jeoongp

**Comment by jdonjdon** [comment]
> > > 
> > 
> > 
> > Oh, are you expecting pwrmgr design change for reset domain issue?
> 
> yes sorry. This change will impact rstmgr / pwrmgr / pinmux. The new ndm reset will have to behave more like a hw_reset, and that will probably need some changes on `pwrmgr` side.
> 
> @jeoongp

Good to know. Please refer this PR when the issue is closed.

**Pull request merged**